cwf/gateway/integ_tests/gy_enforcement_test.go

```diff
@@ -447,19 +447,17 @@ func TestGyCreditExhaustionRedirect(t *testing.T) {
 
 	// we need to generate over 100% of the quota to trigger a session redirection
 	req := &cwfprotos.GenTrafficRequest{
-		Imsi:   imsi,
-		Volume: &wrappers.StringValue{Value: "10M"},
-		//Bitrate: &wrappers.StringValue{Value: "100M"},
+		Imsi:    imsi,
+		Volume:  &wrappers.StringValue{Value: "2M"},
 		Timeout: 60,
 	}
 
-	//time.Sleep(500 * time.Microsecond)
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)
 
 	// Check that enforcement stats flow was not removed and data was passed
 	assert.Eventually(t,
-		tr.WaitForEnforcementStatsForRuleGreaterThan(imsi, "static-pass-all-ocs2", 3*MegaBytes), time.Minute, 2*time.Second)
+		tr.WaitForEnforcementStatsForRuleGreaterThan(imsi, "static-pass-all-ocs2", 1*MegaBytes), time.Minute, 2*time.Second)
 	// Wait for service deactivation
 	assert.Eventually(t,
 		tr.WaitForEnforcementStatsForRule(imsi, "redirect"), time.Minute, 2*time.Second)
```

Comment:
Did you check if the test fails when you generate less than 1M traffic?

Reply:
Yes, this does fail:

```
    gy_enforcement_test.go:461: 
        	Error Trace:	gy_enforcement_test.go:461
        	Error:      	Condition never satisfied
        	Test:       	TestGyCreditExhaustionRedirect
```